iModelCore/libsrc/openssl/BeOpenSSL.mke

```diff
@@ -2359,8 +2357,15 @@ DLM_CONTEXT_LOCATION        = $(BuildContext)Delivery/
 DLM_LIB_CONTEXT_LOCATION    = $(BuildContext)Delivery/
 DLM_CREATE_LIB_CONTEXT_LINK = 1
 
-$(DLM_OBJECT_DEST)$(DLM_NAME).def : $(_MakeFilePath)BeOpenSSL.def
-    $(LinkFirstDepToFirstTarget)
+%if defined(iTwinNativeThirdParty)
+  XC_OPT=\/Y
+  always:
+    xcopy $(_MakeFilePath)BeOpenSSL.def $(DLM_OBJECT_DEST) $(XC_OPT)
+    $(BBPYTHONCMD) $(bsiScripts)RegexReplaceInFile.py iTwinOpenSSL BeOpenSSLB02 $(DLM_OBJECT_DEST)BeOpenSSL.def

```

Comment:
I'm betting xcopy won't fly on non-windows. I'm not sure what the right replacement is.

Reply:
$(copyCmd) is what we should use.

Reply:
Also, instead of "always" let's use a bmake rule:

$(DLM_OBJECT_DEST)BeOpenSSL.def : $(_MakeFilePath)BeOpenSSL.def 
    $(copyCmd) $< $@
    $(BBPYTHONCMD) $(bsiScripts)RegexReplaceInFile.py iTwinOpenSSL BeOpenSSLB02 $@

Reply:
Should I create a separate python script for copying? I am assuming python should work fine on other platforms.

Reply:
I just refresh my page and saw the follow-up comments :)

Reply:
> Also, instead of "always" let's use a bmake rule:
> 
> `$(DLM_OBJECT_DEST)BeOpenSSL.def : $(_MakeFilePath)BeOpenSSL.def ` ` $(copyCmd) $< $@` ` $(BBPYTHONCMD) $(bsiScripts)RegexReplaceInFile.py iTwinOpenSSL BeOpenSSLB02 $@`

Let me add this change and build it again.

Reply:
> `$(DLM_OBJECT_DEST)BeOpenSSL.def : $(_MakeFilePath)BeOpenSSL.def `
> ` $(copyCmd) $< $@`
> ` $(BBPYTHONCMD) $(bsiScripts)RegexReplaceInFile.py iTwinOpenSSL BeOpenSSLB02 $@`

you mean we should just add a condition that if its a windows platform, only then this modification logic for `.def` file should be executed?

Reply:
If it's done this way, you don't need to do any special casing for Windows so I'd just leave it.